### PR TITLE
doc: 2023-04-25 minutes

### LINKED ISSUE #1058: OpenJS Foundation Cross Project Council Meeting 2023-04-25
## Time

**UTC Tue 25-Apr-2023 18:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 25-Apr-2023 11:00 (11:00 AM) |
| US / Mountain | Tue 25-Apr-2023 12:00 (12:00 PM) |
| US / Central  | Tue 25-Apr-2023 13:00 (01:00 PM) |
| US / Eastern  | Tue 25-Apr-2023 14:00 (02:00 PM) |
| EU / Western  | Tue 25-Apr-2023 19:00 (07:00 PM) |
| EU / Central  | Tue 25-Apr-2023 20:00 (08:00 PM) |
| EU / Eastern  | Tue 25-Apr-2023 21:00 (09:00 PM) |
| Moscow        | Tue 25-Apr-2023 21:00 (09:00 PM) |
| Chennai       | Tue 25-Apr-2023 23:30 (11:30 PM) |
| Hangzhou      | Wed 26-Apr-2023 02:00 (02:00 AM) |
| Tokyo         | Wed 26-Apr-2023 03:00 (03:00 AM) |
| Sydney        | Wed 26-Apr-2023 04:00 (04:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Cross%20Project%20Council+Meeting+2023-04-25&iso=20230425T1800
* or https://www.wolframalpha.com/input/?i=06PM+UTC%2C+Apr+25%2C+2023+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1D-PRNE1fy5zoR1uQG-B-eXer7J7REYY1GQCPdyfu8QQ/edit>

## Agenda

Extracted from **cross-project-council-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/cross-project-council

* Simplify CPC Board representation [#1055](https://github.com/openjs-foundation/cross-project-council/pull/1055)
* Open Up Community Board Seat [#1054](https://github.com/openjs-foundation/cross-project-council/issues/1054)
* Should we have a co-chair? [#1043](https://github.com/openjs-foundation/cross-project-council/issues/1043)
* CPC Elections: CPC Director & CPC Chairperson  [#1041](https://github.com/openjs-foundation/cross-project-council/issues/1041)
* Specify how/where to identify individuals from a project to join the CPC [#1026](https://github.com/openjs-foundation/cross-project-council/issues/1026)
* Enabling Payments for GitHub Codespaces for Projects [#1009](https://github.com/openjs-foundation/cross-project-council/issues/1009)
* Create some documentation / web page to describe the collaboration model of OpenJS projects [#1003](https://github.com/openjs-foundation/cross-project-council/issues/1003)
* Update Contributor Covenant Code of Conduct to v2.1 [#851](https://github.com/openjs-foundation/cross-project-council/pull/851)
* Licence check support/tooling [#815](https://github.com/openjs-foundation/cross-project-council/issues/815)
* Clarify requirements around CoC violation [#786](https://github.com/openjs-foundation/cross-project-council/issues/786)

### openjs-foundation/standards

* Secure the Web Forward Workshop - Call for Papers [#213](https://github.com/openjs-foundation/standards/issues/213)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `cross-project-council-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: <https://zoom.us/j/920010234>

* For those who just want to watch: <https://livestream.openjsf.org>

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by joesepi**
closes #1058

### CODE REVIEW

File: meetings/2023/2023-04-25.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  * Joe and Ben will work on this; Ben is already cataloging lead maintainers and project status
+
+* Enabling Payments for GitHub Codespaces for Projects [#1009](https://github.com/openjs-foundation/cross-project-council/issues/1009)
+  * Ben will monitor costs and usage and provide updates. Closing issue
+
+* Create some documentation / web page to describe the collaboration model of OpenJS projects [#1003](https://github.com/openjs-foundation/cross-project-council/issues/1003)
+  * no update
+
+* Update Contributor Covenant Code of Conduct to v2.1 [#851](https://github.com/openjs-foundation/cross-project-council/pull/851)
+  * we can close this PR once patching work is complete, which it nearly is
```
Comment (anonrig) [inline meetings/2023/2023-04-25.md:60]: We need to open another pull request to revert some things in the `.patch` file.

File: meetings/2023/2023-04-25.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+* Joe Sepi (@joesepi)
+* Abby Cabunoc Mayes (@abbycabs)
+* Ben Sternthal (@bensternthal)
+* Ruy Adorno (@ruyadorno)
+* Yagiz Nizipli (@anonrig)
+* Tobie Langel (@tobie)
+* Christian Bromann <@christian-bromann)
+* Michael Dawson (@mhdawson)
+* Tierney Cyren (@bnb)
+
```
Comment (Trott) [inline meetings/2023/2023-04-25.md:19]: I was very, very late, but I was in attendance. It's OK if it doesn't count--I don't know what the rules and norms are around attending only part of the meeting and being marked as present in the minutes--but if it was an oversight....

```suggestion
* Rich Trott (@Trott)

```

Comment (tobie) [reply meetings/2023/2023-04-25.md:20]: > I don't know what the rules and norms are around attending only part of the meeting and being marked as present in the minutes

I'd argue that if you make it on time to put your name in the doc, you should be good, so essentially showing up is fine. Also we don't really have or enforce a policy around good standing.

File: meetings/2023/2023-04-25.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+## Present
+
+* Joe Sepi (@joesepi)
+* Abby Cabunoc Mayes (@abbycabs)
+* Ben Sternthal (@bensternthal)
+* Ruy Adorno (@ruyadorno)
+* Yagiz Nizipli (@anonrig)
+* Tobie Langel (@tobie)
+* Christian Bromann <@christian-bromann)
```
Comment (VoltrexKeyva) [inline meetings/2023/2023-04-25.md:16]:
```suggestion
* Christian Bromann (@christian-bromann)
```

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**